### PR TITLE
Feature/related posts

### PR DESCRIPTION
**Pull request by leezaweb**
I added code to show posts from the same series when there were no posts for the same tag.
There were no Related Articles for the page at '/articles/preventing-and-responding-to-high-risk-disease-outbreaks/'

In Zeplin, the date and themes list do not show for the Next Article on desktop view, but they appear on the mobile view. I deviated from BEM a bit to make the sass shorter--abstract some of the typography without making more placeholders.

### CODE REVIEW

**Comment by leezaweb** [comment]
I noticed another case where no related articles were generated:/events/health-security-in-a-disordered-world/

I developed some conditions, but can't get my where expressions to work out:
`      {% assign related = next_posts | where_exp:"item","item.url != page.url" | sort: date | slice: 1,2 %}
`

File: assets/_sass/components/_post-related-posts.scss [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    }
+  }
+
+  .next-by-theme {
+    background-color: $color-dust;
+    padding: 1.5rem $size-wrapper-padding-mobile;
+    flex-basis: 100%;
+
+    @include breakpoint(medium) {
+      padding: 1.5rem 2.5rem 3rem $size-page-content-padding;
```
Comment (jnschrag) [inline assets/_sass/components/_post-related-posts.scss]: Let's make the padding 2rem on all sides except for the left.

File: assets/_sass/components/_post-related-posts.scss [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    }
+  }
+
+  .related-by-tag {
+    background-color: $color-off-white;
+    padding: 1.5rem $size-wrapper-padding-mobile;
+    flex-basis: 100%;
+
+    @include breakpoint(medium) {
+      padding: 1.5rem $size-page-content-padding 3rem 2.5rem;
```
Comment (jnschrag) [inline assets/_sass/components/_post-related-posts.scss]: Let's make the padding 2rem on all sides.

File: _includes/post-related-posts.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{% if page.collection == 'posts' or page.collection == 'events' %}
+<div class="post__related-posts">
+  {% assign previous_posts = "" | split: ',' %}
+  {% assign next_posts = "" | split: ',' %}
+  {% for theme in page.themes %}
+    {% assign posts = site.posts | concat: site.events %}
+      {% for post in posts %}
+        {% if post.themes contains theme %}
```
Comment (jnschrag) [inline _includes/post-related-posts.html]: This if/elseif block can be rewritten as a single if/else statement

**Pull request merged**